src/sentry/api/bases/organization.py

```diff
@@ -284,80 +284,91 @@ def get_projects(
         standardize how this is used and remove this parameter.
         :param project_ids: Projects if they were passed via request
         data instead of get params
+        :param project_slugs: Project slugs if they were passed via request
+        data instead of get params
         :return: A list of Project objects, or raises PermissionDenied.
+
+        NOTE: If both project_ids and project_slugs are passed, we will default
+        to fetching projects via project_id list.
```

Comment:
Should passing both sets of parameters be an error?

Reply:
🤷 Should it?

I can add that in if that's what we want.